src/js/main.js

```diff
@@ -25,3 +25,6 @@ document.addEventListener("click", (event) => {
 document.querySelector(".site-nav .nav-smallScreen .search-container svg").addEventListener("click", () => {
 	document.querySelector(".site-nav .nav-smallScreen .search-container input").focus();
 });
+
+var h1 = document.getElementById("project-we-count-evaluates-toronto's-quayside-master-innovation-and-development-plan");
```

Comment:
I don't think this type of specific handling by matching a particular text content is acceptable. But I see your point. I will update the corresponding WordPress post to try your approach. Thanks.

Reply:
> I don't think this type of specific handling by matching a particular text content is acceptable. But I see your point. I will update the corresponding WordPress post to try your approach. Thanks.


Because this bug only fixed by this way or from wordpress site